hack/get_ci_vm.sh

```diff
@@ -13,16 +13,42 @@ DISK="200"
 PROJECT="libpod-218412"
 GOSRC="/var/tmp/go/src/github.com/containers/libpod"
 
-PGCLOUD="sudo podman run -it --rm -e AS_ID=$UID -e AS_USER=$USER -v /home/$USER:$HOME:z quay.io/cevich/gcloud_centos:latest"
+PGCLOUD="sudo podman run -it --rm -e AS_ID=$UID -e AS_USER=$USER -v /home/$USER:$HOME:z -v /tmp:/tmp:ro quay.io/cevich/gcloud_centos:latest"
```

Comment:
The `:z` on this line is definitely WRONG.  This will cause your entire homedir to be mislabeled.

Remove the :z and switch to using --security-option lable=disabled

To turn off SELinux separation.

Reply:
Oooohhh, ok that sounds like a much better idea.  Thanks!